src/pages/studyView/StudyViewUtils.tsx

```diff
@@ -817,6 +822,28 @@ export function getUniqueKey(attribute: ClinicalAttribute): string {
     return attribute.clinicalAttributeId;
 }
 
+export function getUniqueNamespaceKey(attribute: NamespaceAttribute): string {
+    return attribute.outerKey.concat('_', attribute.innerKey);
+}
+
+function getNamespaceChartDisplayName(
+    attribute: NamespaceAttribute,
+    namespaceAttributes: NamespaceAttribute[]
+): string {
+    var count = 0;
+    namespaceAttributes.forEach(item => {
+        if (item.innerKey === attribute.innerKey) {
+            count++;
+        }
+    });
+
+    if (count > 1) {
+        return attribute.innerKey + ' (' + attribute.outerKey + ')';
```

Comment:
not a big deal but you should use js string interpolation to make string building a little more clear.